(googlechrome) Added revision padding

AdmiringWorm opened the pull request:
This will add padding to the revision number whenever the version is less than `56.0.2925`

**IMPORTANT merge using `[AU googlechrome]`, otherwise the padding is applied but the revision number is not incremented (but still be pushed since `56.0.2925.87 != 56.0.2925.87000`**

fixes #604

pascalberger commented on automatic/GoogleChrome/update.ps1:
```diff
@@ -1,6 +1,8 @@
+. "$PSScriptRoot\..\..\scripts\Get-Padded-Version.ps1"
 
 $releases = 'http://omahaproxy.appspot.com/all?os=win&amp;channel=stable'
+$paddedUnderVersion = '56.0.2925'
```
Wouldn't it be better to have this also for future versions (assuming that Google won't change its practice)?

AdmiringWorm replied:
I'm not sure, which is why I opened the PR instead of committing directly

ferventcoder replied:
I like staying on the side of safety for now. If we need to do this later it is easy enough to add back in.

AdmiringWorm replied:
👍 I'll update again to add it back

ferventcoder commented:
Why four `0000` after the number? Wouldn't two or three be sufficient? If you go 1,000 package revisions, I think there is a problem.

AdmiringWorm commented:
@ferventcoder 
The reason I've been telling myself is to try showing a difference between the official version and the package fix iteration. Considering the next released revision version would be 1xx I don't think it would be too many zeroes.

AdmiringWorm commented:
@pascalberger I've updated to add padding for all versions
@ferventcoder I've changed the revision length to be a maximum length of 5 (making it 1-2 zeroes between actual version and fix incrementation)

ferventcoder commented:
I actually liked not having the padding until we need it, so I liked erring on the side of safety.

AdmiringWorm commented:
@ferventcoder I just readded the `paddedUnderVersion` variable.

AdmiringWorm commented:
looks like google chrome is failing all checks.
I'm guessing because of aws outage?
Guess we need to repush when the outage is over.
